plugins/keepconf/keepconf_time

```diff
@@ -0,0 +1,99 @@
+#!/bin/sh
+# -*- sh -*-
+
+: << =cut
+
+=head1 NAME
+
+keepconf_time -Indicate the times of keepconf execution
+
+=head1 APPLICABLE SYSTEMS
+Linux box with keepconf installed.
+
+=head1 CONFIGURATION
+
+[keepconf_*]
+env.files /var/tmp/keepconf-mon.csv
+# Multifile its available
+#env.files /var/tmp/keepconf-client1.csv /var/tmp/keepconf-client2.csv
+
+
+=head1 AUTHOR
+
+Ricardo F. <rfrail3@yahoo.es>
+
+=head1 LICENSE
+
+GPLv2
+
+=head1 VERSION
+v.1.2.0
+
+=head1 MAGICK MARKERS
+
+ #%# family=auto
```

Comment:
Since both plugins share most of their code, it could be a good approach to combine both.
In this case you would add the "suggest" capability returning "status" and "time".